uint8 playerNum
uint8 teamNum
uint8 fallen
float32[3] pose
float32 ballAge
float32[2] ball
